pkgs/tools/misc/optinix/default.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  fetchFromGitLab,
+  buildGoModule,
+  installShellFiles,
+}:
+buildGoModule rec {
+  pname = "optinix";
+  version = "0.1.1";
+
+  src = fetchFromGitLab {
+    owner = "hmajid2301";
+    repo = "optinix";
+    rev = "v${version}";
+    sha256 = "sha256-bRHesc03jExIL29BCP93cMbx+BOT4sHCu58JjpmRaeA=";
+  };
+
+  vendorHash = "sha256-uSFEhRWvJ83RGpekPJL9MOYJy2NfgVdZxuaNUMq3VaE=";
+
+  nativeBuildInputs = [
+    installShellFiles
+  ];
+
+  postInstall = ''
+    installShellCompletion --cmd optinix \
+      --bash <($out/bin/optinix completion bash) \
+      --fish <($out/bin/optinix completion fish) \
+      --zsh <($out/bin/optinix completion zsh)
+  '';
+
+  meta = with lib; {
+    description = "A CLI tool for searching options in Nix, written in Go, powered by the bubbletea framework for TUI.";
```

According to [pkgs/README.md](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes) `meta.description` should be short and factual, mentioning the implementation language and used TUI framework feels a bit too verbose.
If you believe that is important information, please consider adding `meta.longDescription`:
```suggestion
    description = "Tool for searching options in Nix";
```

---

fixed